WeakAuras/Prototypes.lua

```diff
@@ -6464,6 +6464,63 @@ Private.event_prototypes = {
     automaticrequired = true,
     statesParameter = "one",
   },
+  ["Loot Specialization"] = {
+    type = "unit",
+    events = function()
+      local events = {"PLAYER_LOOT_SPEC_UPDATED", "PLAYER_SPECIALIZATION_CHANGED" }
+      return {
+        ["events"] = events
+      }
+    end,
+    force_events = "PLAYER_LOOT_SPEC_UPDATED",
+    name = L["Loot Specialization"],
+    init = function(trigger)
+      return [[
+         local lootSpecId = GetLootSpecialization()
+         local currentSpecId = GetSpecializationInfo(GetSpecialization())
+         if lootSpecId == 0 then --Player chose 'Current Specialization'
+          lootSpecId = currentSpecId
+         end
+         local _, lootSpecName, _, lootSpecIcon = GetSpecializationInfoByID(lootSpecId)
+      ]]
+    end,
+    args = {
+      {
+        name = "name",
+        display = L["Loot Specialization"],
+        type = "multiselect",
+        init = "lootSpecId",
```

Comment:
This stores the lootSpecId, since that's needed for the test. I suggest changing the name of this to "lootSpecId" and adjusting the "lootSpecId" below to be actually initialized by lootSpecName and be named "name".